pkg/server/server.go

```diff
@@ -154,12 +158,20 @@ func (s *Server) Start(errCh chan error) error {
 		}()
 	}
 
+	// Start build poller
+	log.Info("Starting builds poller")
+	err = s.BuildPoller.Start()
```

Comment:
I think this should be started in the `serve` command and not by the server itself. It would make it more flexible in the future.